chef/api.py

```diff
@@ -70,6 +56,8 @@ class ChefAPI(object):
     env_value_re = re.compile(r'ENV\[(.+)\]')
     ruby_string_re = re.compile(r'^\s*(["\'])(.*?)\1\s*$')
 
+    verify_ssl = True
```

Comment:
Please set the in the initialize. Using class MRO callback for default instance values is a really bad pattern.